website/src/pages/index.js

```diff
@@ -116,7 +116,7 @@ function ActorExample() {
                     The fastest way to try Crawlee out is to use the <b>Crawlee CLI</b> and choose one of the provided templates. The CLI will prepare a new project for you, and add boilerplate code for you to play with.
                 </p>
                 <CodeBlock className="language-bash">
-                    pipx run crawlee create my-crawler
+                    pipx run crawlee[cli] create my-crawler
```

Comment:
please add single quotes, because of zsh

```suggestion
                    pipx run 'crawlee[cli]' create my-crawler
```